setup.py

```diff
@@ -41,11 +41,11 @@
     'deform-jinja2>=0.5,<0.6',
     'elasticsearch>=1.1.0,<2.0.0',
     'gevent>=1.0.2,<1.1.0',
-    'gnsq>=0.3.0,<0.4.0',
     'gunicorn>=19.2,<20',
     'itsdangerous>=0.24',
     'jsonpointer==1.0',
     'jsonschema>=2.5.1,<2.6',
+    'kombu>=3.0.35',
```

Comment:
As with the other version specifications in this file, it's probably wise to add an upper bound on the assumption that the project is semantically versioned, so `kombu>=3.0.35,<3.1`.